portfolio/src/main/webapp/style.css

```diff
@@ -6,4 +10,40 @@
 
 #greeting-container {
   margin-top: 20px;
+  font-family: Arial;
+  color: chartreuse;
+}
+
+#img-selfie{
+    /* display: block;
+    margin-left: auto;
+    margin-right: auto; */
+    width: 30%;
+    height: 30%;
+}
+
+/* CLASS SELECTORS */
+
+.header{
+    background-color: red;
 }
+
+/* TYPE SELECTORS */
+
+body{
+    background-color: black;
+    color: white;
+    text-align: right;
+}
+
+h1{
```

Comment:
Why not set the font family for the whole document instead of re-defining it 3 times?

Reply:
Fixed!